### ISSUE TITLE
Minor issues with screenreader mode

### ISSUE THREAD

**Issue by bkirwi** [andrewschultz/you-wont-get-her-back#29]
If you turn screenreader mode on, the first description is:

> **b6**
> You're currently at b6. Your ally, the white rook, is at nothing.
>  
> The enemy king is still skulking at a1, watching the carnage from afar. The enemy rook is at d5.
> .

Two issues you'll notice:
- It mentions 'the white rook', but there's no such piece. In fact, if you `look` again, it will choose another piece that doesn't exist seemingly at random. And it's described as being "at nothing" which is accurate but unhelpful. Plus the pawn is missing!
- There's a trailing period.

I otherwise quite appreciate the screenreader mode, by the way! (I don't use a screenreader; I just like the prose.)

**Comment by andrewschultz** [comment]
Thanks for this bug report! I saw what was happening pretty quickly, and it's a bit embarrassing, but it's a quick fix.

I should have tested screen reader mode more heavily.

Here is the code:

```
to say text-board-description:
	let my-piece be a random allied person;
	say "You're currently at [location of player]. Your ally, [the my-piece], is at [location of my-piece].";
```

However, I'd defined "allied" as anyone potentially on your side, even if they aren't on the board. Defining "writeoutable" as "allied, but not off-stage" should fix that.

When I did a brief check to make sure, I saw the pawn listed, as it appeared by random. So I thought "Ah! Great! This works!" But really I'd just gotten lucky.

This is important enough I may send in an update before the comp ends. However, if I don't, it'll really push me to do a quick post-comp release.
